src/Identity/ApiAuthorization.IdentityServer/src/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.csproj

```diff
@@ -5,19 +5,29 @@
     <Description>ASP.NET Core API Authorization package powered by Identity Server.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;apiauth;identity</PackageTags>
+    <HasReferenceAssembly>false</HasReferenceAssembly>
     <IsShippingPackage>true</IsShippingPackage>
+    <HasReferenceAssembly>false</HasReferenceAssembly>
```

Comment:
Set this once (dupe of line 8). As long as our policies allow us to live w/o a ref/ assembly, wonderful 😃